playbooks/os_updates.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: all
+  serial: 2
+  remote_user: pulsys
+  become: true
+  pre_tasks:
+
+    - name: test for python on new VM
+      ansible.builtin.raw: test -e /usr/bin/python
+      changed_when: false
+      failed_when: false
+      register: check_python
+
+    - name: Install Python  # this has to be raw in the event python is not installed
+      ansible.builtin.raw: apt -y update && apt install -y python-minimal python-pip
+      when: check_python.rc != 0
+
+  tasks:
+
+    - name: Update apt
```

Comment:
Combine this task and the one below - `apt-get dist-upgrade` also upgrades packages - it's a twofer!